## SquareDataMessage
# No Documentation Annotation Found

geometry_msgs/Vector3 location

geometry_msgs/Quaternion rotation

float64 lengthX

float64 widthY


